Svc/ActiveLogger/docs/sdd.md

```diff
@@ -94,11 +94,11 @@ TBD
 
 ## 5. Module Checklists
 
-Document | Link
--------- | ----
-Design Checklist | [Link](Checklist_Design.xlsx)
-Code Checklist | [Link](Checklist_Code.xlsx)
-Unit Test Checklist | [Link](Checklist_Unit_Test.xls)
+Checklist |
+-------- |
+[Design](Checklist_Design.xlsx) |
+[Code](Checklist_Code.xlsx) |
+[Unit Test](Checklist_Unit_Test.xls) |
```

Comment:
There are a couple of variations on these checklists. I don't think the two column format helps anyone, it certainly hurts anyone using a screen reader (and it also hides bugs ...).